cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/SparkCqlEvaluator.java

```diff
@@ -532,6 +542,13 @@ protected ContextDefinitions readContextDefinitions(String path) throws Exceptio
             String dataType = (String) datarow.getValue(ContextRetriever.SOURCE_FACT_IDX);
             List<Object> mappedRows = dataByDataType.computeIfAbsent(dataType, x -> new ArrayList<>());
             mappedRows.add(datarow);
+
+            if (dataTypeAliases.containsKey(dataType)) {
+                String mappedType = dataTypeAliases.get(dataType);
+
+                mappedRows = dataByDataType.computeIfAbsent(mappedType, x -> new ArrayList<>());
```

Comment:
Pedantic: The re-assigning of `mappedRows` makes me a little bit nervous.
I could see a future dev accidentally referring to the list outside the `if` block expecting it to be the data's actual list rather than it's abstract list.

Reply:
addressed in b6ea8c2